emoji-androidx-emoji2/src/commonMain/kotlin/com/vanniktech/emoji/androidxemoji2/AndroidxEmoji2.kt

```diff
@@ -40,7 +40,7 @@ internal class AndroidxEmoji2 internal constructor(
 
   override fun equals(other: Any?): Boolean {
     if (this === other) return true
-    if (other == null || this::class != other::class) return false
```

Comment:
Could you update your branch with the latest changes from master?

Reply:
Looks like #1106 didn't include the update to the template.

Reply:
Ah, my bad. Then never mind! Thanks for the change